Nick03022019

NickJacksonDev opened the pull request:
IMPORTANT: 
will probably need to fix the .gitignore file since Ben and I both edited it
may still have problem with the Campaign Manager in installed apps

Elocutioner committed:
Update settings.cpython-37.pyc

Elocutioner committed:
got rid of duplicate setting

campaign manager was in installed apps twice

Elocutioner committed:
Merge branch 'Nick03022019' of https://github.com/NickJacksonDev/DnD-Manager into Nick03022019

Elocutioner committed:
Merge branch 'master' into Nick03022019

SpinksJared committed:
Added Database query printouts

Elocutioner committed:
Update settings.py

SpinksJared committed:
Merge branch 'Nick03022019' of https://github.com/NickJacksonDev/DnD-Manager into Nick03022019

Elocutioner committed:
Update .gitignore

Elocutioner committed:
Can now add campaigns to the DB via the site

DelgadoJosh commented:
Looks good to me. The html for the campaign manager looks good. I haven't opened up the page on my side, but the code looks good

brhiner commented on Campaign_Manager/forms.py:
```diff
@@ -0,0 +1,7 @@
+from django import forms
+from .models import Campaign
+
+class CampaignForm(forms.ModelForm):
```
Campaign form is a little vague. Maybe CreateCampaignForm?

SpinksJared commented:
From what I can tell it looks good to me.

brhiner commented on Campaign_Manager/templates/Campaign_Manager/campaign_manager-home.html:
```diff
@@ -1,4 +1,26 @@
+
+    {% endfor %}
+
+    {% for Character in characters %}
```
As discussed offline, we can remove this from this file

brhiner reviewed (changes requested):
Code looks good. I've posted a few questions/reminders below to address before you're good to merge
- Don't forget to pull from master and resolve any conflicts before submitting
- Remove DND_Manager/__pycache__/urls.cpython-37.pyc from your commit

Elocutioner committed:
Updated html and Campaign Form

Changed CampaignForm to CreateCampaignForm
Removed Character List from Campaign Manager page.

Elocutioner committed:
Update from master

NickJacksonDev commented:
I don't really understand how to edit a merge request once it has been initiated so I am just making a new one with all of Ben's concerns addressed.
